tests/test_step_parsing.py

```diff
@@ -139,3 +141,12 @@ def test_input_extras(input_extras_config):
 ])
 def test_input_keep_directories(value, expected):
     assert Input(name="foo", keep_directories=value).keep_directories == expected
+
+
+def test_timeouts(timeouts_config):
+    assert timeouts_config.steps['short-time-limit'].time_limit == datetime.timedelta(seconds=300)
+    assert timeouts_config.steps['short-time-limit'].no_output_timeout is None
+    assert timeouts_config.steps['big-no-output-timeout'].time_limit is None
+    assert timeouts_config.steps['big-no-output-timeout'].no_output_timeout.days == 1
+    assert timeouts_config.steps['human-readable-time-limit'].time_limit == datetime.timedelta(seconds=5405)
+    assert timeouts_config.steps['human-readable-time-limit'].no_output_timeout.total_seconds() == 86400 * 2
```

Comment:
Looking at the test fixture and the test, I took a double take at how 
```
- step:
    name: human-readable-time-limit
    image: busybox
    command: foo {parameters}
    time-limit: 1h 30m 5s
    no-output-timeout: 48h
```
leads to 
`assert timeouts_config.steps['human-readable-time-limit'].no_output_timeout.total_seconds() == 86400 * 2`
especially since we mix `total_seconds`, `days` and just the vanilla timedelta in these checks. Is it purposeful to have these different approaches or might it make it more readable 
`...time_limit == datetime.timedelta(seconds=300)`
way in all of these?

Reply:
Sure. :)